packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -1339,6 +1339,20 @@ class FileCollection extends DocumentCollection {
 
     return { newLastSeq, pending, results }
   }
+
+  /**
+   * Get the Shared Drives directory or create it if it does not exist.
+   *
+   * The Shared Drives directory is a special directory :
+   * - its _id is io.cozy.files.shared-drives-dir
+   * - it contains Nextcloud drives
+   * - it contains shared drives
+   *
+   * @returns {IOCozyFolder} Shared Drives directory
+   */
+  getSharedDrivesDirectoryOrCreate() {
```

Comment:
I would prefer `getOrCreateSharedDrivesDirectory` , which is a more consistent naming, and I think less misleading, as the `create` at the name could easily be missed.
If yes, is it documented somewhere? This is not very intuitive from my point of view

Reply:
Agree about renaming to `getOrCreateSharedDrivesDirectory`. I do not understand the last part of your comment ?

Reply:
The last part of the comment does not make any sense, because I somehow mixed up my comments. This was meant to be here: https://github.com/cozy/cozy-client/pull/1590#discussion_r1967603296